src/mlpack/bindings/R/mlpack/src/r_util.cpp

```diff
@@ -117,16 +117,16 @@ void IO_SetParamRow(const std::string& paramName,
 // Call IO::SetParam<arma::Row<size_t>>().
 // [[Rcpp::export]]
 void IO_SetParamURow(const std::string& paramName,
-                      const arma::Row<size_t>& paramValue)
+                     const arma::Row<size_t>& paramValue)
 {
-  IO::GetParam<arma::Row<size_t>>(paramName) = std::move(paramValue);
+  IO::GetParam<arma::Row<size_t>>(paramName) = std::move(paramValue - 1);
```

Comment:
```suggestion
  IO::GetParam<arma::Row<size_t>>(paramName) = paramValue - 1;
```
Since we're modifying the input, the `std::move()` isn't needed here.

Reply:
Done.